plugins/module/create_service_account.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: create_service_account
+
+short_description: Create a Service Account for use with Red Hat Openshift Application Services 
+
+# If this is part of a collection, you need to use semantic versioning,
+# i.e. the version is of the form "2.5.0" and not "2.4".
+version_added: "0.1.0"
+
+description: Create a Service Account for use with Red Hat Openshift Application Services 
+
+options:
+    name:
+        description: Name of the service account
+        required: true
+        type: str
+    description:
+        description: Description of the service account
+        required: true
+        type: str
+ 
+# Specify this value according to your collection
+# in format of namespace.collection.doc_fragment_name
+extends_documentation_fragment:
+    - redhat.rhoask.my_doc_fragment_name
```

Comment:
?

Reply:
I don't have doc fragments in there yet. I'm going down the route of autodoc gen as per your suggestion, I just need to perfect it, complete docs should be up in a PR soon